diff --git a/lib/Authen/Krb5.xs b/lib/Authen/Krb5.xs
index 3fa1e91..090782e 100644
--- a/lib/Authen/Krb5.xs
+++ b/lib/Authen/Krb5.xs
@@ -650,0 +651,10 @@ krb5_recvauth(auth_context,fh,version,server,keytab)
+void
+cc_copy_creds(incc,outcc)
+	Authen::Krb5::Ccache incc
+	Authen::Krb5::Ccache outcc
+
+	CODE:
+	err = krb5_cc_copy_creds(context, incc, outcc);
+	if (err) XSRETURN_UNDEF;
+	XSRETURN_YES;
+
